spec/support/features/doc_auth_helper.rb

```diff
@@ -116,6 +116,13 @@ def complete_document_capture_step
     attach_and_submit_images
   end
 
+  # yml_file example: 'spec/fixtures/puerto_rico_resident.yml'
+  def complete_document_capture_step_with_yml(yml_file)
+    attach_file 'Front of your ID', File.expand_path(yml_file)
+    attach_file 'Back of your ID', File.expand_path(yml_file)
+    click_on 'Submit'
```

Comment:
I think we should use the `t` helper here to avoid duplicating the source of truth for UI strings (and in the off-chance we'd ever want to run tests in a language other than English).

Reply:
This yml_file is a proofing yml for document capture. Were you thinking it was a translation file, or is there a way to use `t` helpers to find proofing ymls? I can update the name.

Reply:
I believe Andrew is referring to `.* of your ID` and `Submit` here.

Reply:
Oh right, the `Submit` too!

Reply:
Yes, meaning to swap the hard-coded "Front of your ID", "Back of your ID", and "Submit" labels with whatever the equivalent `t` call would be.

Reply:
Good idea, did that here, and in a few more places. Also this reminded me to call this helper method in the test I originally copied the code from!